websites/M/MDN Web Docs/presence.ts

```diff
@@ -0,0 +1,102 @@
+const presence = new Presence({
+    clientId: "786770326234464256"
+}), stamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const data: PresenceData = {
+        startTimestamp: stamp
+    }, paths = document.location.pathname.split('/');
+    paths.splice(0, 1);
```

Comment:
this is unnecessary lol, you still use paths[1] to get paths, so why you remove this blank space? You only make this presence longer for no reason

Reply:
okaaay